mParticle-Kochava.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
     s.name             = "mParticle-Kochava"
-    s.version          = "8.2.0"
+    s.version          = "9.0.0"
```

Comment:
This should be 8.3.0, as we'll only bump major version when we go to version 9 of Core.

Reply:
Also it occurs to me that when we start using semantic release CI, we won't update this number manually at all anymore.

Reply:
Oh and actually, we should not even update the version at all in this PR and do it when we make the release like we usually do with a "Update to version 8.3.0" commit...